assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -125,13 +37,12 @@
                         <div class="col-12 col-sm-6 order-1 order-sm-0 p-0 px-sm-2">
                             <a class="btn btn-primary btn-block" href="
                             {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.associationdetails.shorthand %}">
-                                + Add diner</a>
+                                + Add Diner</a>
```

Comment:
Why two capitals instead of the correct 'Add diner'?

Reply:
Somehow looked better. On short two-word sentences on buttons and the like I often use capitals for nouns to make them stand out more. 'diner' feels rather unimportant in it's visual context otherwise. But it was more of a personal style thing. I'm fine either way.